hdfs_namenode/conf.yaml.example

```diff
@@ -10,7 +10,9 @@ instances:
   # the property dfs.http.address or dfs.namenode.http-address
   #
   -  hdfs_namenode_jmx_uri: http://localhost:50070
-
+  #   tags:
```

Comment:
We should add a small description of what `tags` does, like https://github.com/DataDog/integrations-core/blob/master/mapreduce/conf.yaml.example#L23